api/python/provisioner/inputs.py

```diff
@@ -877,3 +879,58 @@ def is_iso(self):
 @attr.s(auto_attribs=True)
 class SWUpgradeRepo(SWUpdateRepo):
     _param_di = param_spec['swupgrade/repo']
+    target_build: str = attr.ib(
+        default=None,
+        metadata={
+            METADATA_ARGPARSER: {
+                'help': 'file path to base directory for SW upgrade'
+            }
+        }
+    )
+
+    @property
+    def pillar_key(self):
+        # the local root pillar key for 'swupgrade' installation type
+        return "repos"
```

Comment:
It means each new upgrade bundle will replace all current ones. Please consider the different scheme:
```yaml
upgrade
  repos:
    sw_upgrade_<RELEASE>:
    sw_upgrade_<RELEASE>_OS:
    ...
```

Actually the more beter options would be to group them per release. But it would require to review our states deeper.

Reply:
Yes, I have already used this scheme in the first version of this PR but without "sw_upgrade_" prefix. In current version of API I group repositories by its version. And added "sw_upgrade_" prefix